pages/dinner.vue

```diff
@@ -1,17 +1,295 @@
 <template>
-  <h1>🍲01.10.2025</h1>
-  <h2>Menu of the day</h2>
-  Tatziki med bagt kartoffel
-  <h2> Kuverter 90</h2>
-  <list-disc>
-    <li> 37% Takeaway: 30 kuverter ... 6 minutter tilbage til deadline></li>
-    <li>63% Spisesal: 50 kuverter</li>
-  </list-disc>
-
-  <h2>Allergier</h2>
-  <list-disc>
-    <li>tomater 2 (anna, lea)</li>
-    <li>hvidløg 8 (anna, lea, mark, thomas, ...)</li>
-    <li>mælk 17 (anna, lea, mark, thomas, jens, jonas, jacob, johanne, ...)</li>
-  </list-disc>
+<div>
+    <!-- Calendar Header -->
+    <div class="flex justify-between items-center mb-4">
```

Comment:
det kunne være fedt hvis calendar er en vue component for sig - så kan vi genbruge den